sdk/healthinsights/health-insights-radiologyinsights-rest/test/snippets.spec.ts

```diff
@@ -13,13 +13,17 @@ import RadiologyInsightsRestClient, {
   FollowupRecommendationInference,
   GenericProcedureRecommendation,
   getLongRunningPoller,
+  GuidanceInference,
   ImagingProcedure,
   ImagingProcedureRecommendation,
   isUnexpected,
   LateralityDiscrepancyInference,
   LimitedOrderDiscrepancyInference,
+  PresentGuidanceInformation,
+  QualityMeasureInference,
   RadiologyInsightsJobOutput,
   RadiologyProcedureInference,
+  ScoringAndAssessmentInference,
```

Comment:
probably should import from "@azure-rest/health-insights-radiologyinsights-rest" in snppets.spec.ts